squire_lib/src/accounts.rs

```diff
@@ -0,0 +1,20 @@
+use uuid::Uuid;
+use settings;
+
+struct squire_account {
+    display_name: String,
+    user_name: String,
+    user_id: Uuid,
+    do_share: bool,
+}
+
+struct organization_account {
+    display_name: String,
+    user_name: String,
+    user_id: Uuid,
+    do_share: bool,
+    owner: squire_account,
+    default_judge: squire_account,
+    admin_account: squire_account,
```

Comment:
Both the judge and admin fields should be `Vec<squire_account>` since an org might have multiple judges as defaults.